bots/elasticsearch/tasks.py

```diff
@@ -78,14 +86,14 @@ def serialize_person(self, person, suggest=True):
                 for affiliation in
                 person.affiliations.all()
             ],
-            'sources': [source.robot for source in person.sources.all()],
+            'sources': [self.serialize_source(source) for source in person.sources.all()],
         }
         return add_suggest(serialized_person) if suggest else serialized_person
 
     def serialize_entity(self, entity, suggest=True):
         serialized_entity = {
-            '@id': entity.pk,
```

Comment:
This is also for human-friendliness, for queries like `type:preprint` instead of `@type:preprint`. It probably doesn't matter for `@id`; I just did that for consistency.